v2/pkg/subscraping/sources/zoomeyeapi/zoomeyeapi.go

```diff
@@ -0,0 +1,80 @@
+package zoomeyeapi
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/projectdiscovery/subfinder/v2/pkg/subscraping"
+)
+
+//type loginResp struct {
```

Comment:
Would it be possible to remove the commented code if not necessary?

Reply:
yes. I copy it from zoomeye and forget to remove some comment.